src/linear_algebra/basis.lean

```diff
@@ -570,6 +570,22 @@ begin
         apply linear_map.map_le_range } } }
 end
 
+lemma le_of_span_le_span {s t u: set β} (zero_ne_one : (0 : α) ≠ 1)
```

Comment:
Do we want `zero_ne_one` as explicit argument, or should typeclass inference try to find it for us?
Also applies to other places...

Reply:
We don't have a `nonzero_ring` type class currently, only `nonzero_comm_ring`

Reply:
But there is `zero_ne_one_class`. Which will relieve the user of this triviality when `α` happens to be a field or a `nonzero_comm_ring` or something like that.

Reply:
But then there's a `has_one` and `has_zero` diamond

Reply:
Blast! Doesn't it have `has_one` and `has_zero` in the assumptions? Never mind. Forget about what I said.